src/Input/input.style.js

```diff
@@ -5,6 +5,7 @@ const InputStyles = styled.div`
   width: 100%;
 
   label {
+    position: relative;
```

Comment:
Make sure this is responsive and looks good on small and large screens

Reply:
done